examples/ipyradiant.ipynb

```diff
@@ -1,17 +1,26 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## ipyradiant Example Widget\n",
+    "The `RadiantObject` widget is an example of a tool that aggregates widgets available within the `ipyradiant` package. "
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# This won't work until we have a package\n",
     "import traitlets as T\n",
     "\n",
     "import ipywidgets as W\n",
-    "from ipyradiant import LoadWidget, QueryWidget\n",
-    "from rdflib import BNode, Graph"
+    "from rdflib import BNode, Graph\n",
+    "\n",
+    "from ipyradiant.loader import LoadWidget\n",
```

Comment:
as mentioned, importing these from `ipyradiant` would be shorter, better... nobody cares about the internal structure